airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeV1V2Migrator.java

```diff
@@ -87,9 +87,11 @@ protected Optional<SnowflakeTableDefinition> getTableIfExists(final String names
   @Override
   protected NamespacedTableName convertToV1RawName(final StreamConfig streamConfig) {
     // The implicit upper-casing happens for this in the SqlGenerator
+    @SuppressWarnings("deprecation")
+    String tableName = this.namingConventionTransformer.getRawTableName(streamConfig.id().originalName());
```

Comment:
getRawTableName is deprecated. I had to move the call to its own line for the SuppressWarnings to be accepted